### PR TITLE
Data update: Change feature layer renderer

### PR DESCRIPTION
**Pull request by ZackAllen**

### CODE REVIEW

**Review by nCastle1 — CHANGES_REQUESTED** [review]
Code changes look good, but the readme details (About the data) will need to be updated.

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/DisplayDeviceLocation/readme.metadata.json [hunk 1]
```diff
@@ -20,8 +20,7 @@
     },
     "offline_data": [],
     "redirect_from": [
-        "/net/latest/wpf/sample-code/displaydevicelocation.htm",
-        "/net/wpf/sample-code/display-device-location/"
+        "/net/latest/wpf/sample-code/displaydevicelocation.htm"
     ],
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/DisplayDeviceLocation/readme.metadata.json:24]: Was this an intentional change?

**Review by nCastle1 — APPROVED** [review]
Approved with one lingering question.

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/DisplayDeviceLocation/readme.metadata.json (hunk 1 above)
Comment (ZackAllen) [reply src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/DisplayDeviceLocation/readme.metadata.json]: Not intentional. I had the display device location issue resolved in my other PR, forgot about it here.

**Pull request merged**